app/build.gradle

```diff
@@ -62,22 +62,20 @@ android {
             minifyEnabled false
         }
     }
-
-    // Allows the use of lambda and method reference syntax.
+     // Allows the use of lambda and method reference syntax.
```

Comment:
We don't actually need this whole block anymore.

Reply:
we are still using lamdas in Ripple, without the block it gives an error

Reply:
Ah cool. Sorry, thought that went away when Android started supporting Java 8. Guess we just don't need RetroLambda anymore, but looks like that is gone.